features/adres.feature

```diff
@@ -0,0 +1,180 @@
+    Gegeven persoon met burgerservicenummer "999990482" heeft de volgende gegevens bij verblijfplaats:
+    """
+      "gemeente van inschrijving": "1681",
+      "straatnaam": "1e Exloërmond",
+      "huisnummer": 3,
+      "aanduidingBijHuisnummer": "tegenover",
+      "postcode": "9573PA"
+    """
+    Als persoon met burgerservicenummer "999990482" wordt opgevraagd met fields=verblijfplaats
+    Dan bevat in het antwoord verblijfplaats:
+    """
+      "straat": "1e Exloërmond",
+      "korteNaam": "1e Exloërmond",
+      "woonplaats": "Borger-Odoorn",
+      "adresregel1": "1e Exloërmond 3 tegenover",
+      "adresregel2": "9573PA Borger-Odoorn"
+    }
+    """
+
+  Scenario: verblijfplaats is een locatiebeschrijving
+    Gegeven persoon met burgerservicenummer "000009921" heeft de volgende gegevens bij verblijfplaats:
+    """
+      "gemeenteVanInschrijving": "0518",
+      "locatiebeschrijving": "Woonboot in de Grote Sloot"
+    """
+    Als persoon met burgerservicenummer "000009921" wordt opgevraagd met fields=verblijfplaats
+    Dan bevat in het antwoord verblijfplaats:
+    """
+      "woonplaats": "'s-Gravenhage",
+      "adresregel1": "Woonboot in de Grote Sloot",
+      "adresregel2": "'s-Gravenhage"
+    }
+    """
+    En bevat het antwoord geen veld verblijfplaats.straat
+    En bevat het antwoord geen veld verblijfplaats.korteNaam
+
+  Scenario: verblijfplaats is een adres in het buitenland
+    Gegeven persoon met burgerservicenummer "000009921" heeft de volgende gegevens bij verblijfplaats:
+    """
+      "adresregel1": "Rue du pomme 25",
+      "adresregel2": "Bruxelles",
+      "adresregel3": "postcode 1000",
+      "land": "5010"
+    """
+    Als persoon met burgerservicenummer "999993653" wordt opgevraagd met fields=verblijfplaats
+    Dan bevat in het antwoord verblijfplaats:
+    """
+      "adresregel1": "Rue du pomme 25",
+      "adresregel2": "Bruxelles",
+      "adresregel3": "postcode 1000",
+      "land": {
+        "code": "5010",
+        "omschrijving": "België"
+      }
+    """
+    En bevat het antwoord geen veld verblijfplaats.straat
+    En bevat het antwoord geen veld verblijfplaats.korteNaam
+    En bevat het antwoord geen veld verblijfplaats.woonplaats
+
+  Scenario: verblijfplaats is onbekend (vertrokken onbekend waarheen)
+    Gegeven persoon met burgerservicenummer "999993586" heeft bij verblijfplaats verblijfbuitenland.land met code "0000"
+    Als persoon met burgerservicenummer "999993586" wordt opgevraagd met fields=verblijfplaats.adresregel1,verblijfplaats.adresregel2,verblijfplaats.adresregel3,verblijfplaats.land,verblijfplaats.vertrokkenOnbekendWaarheen
+    Dan bevat het antwoord verblijfplaats.vertrokkenOnbekendWaarheen met de waarde true
+    En bevat het antwoord geen veld verblijfplaats.adresregel1
+    En bevat het antwoord geen veld verblijfplaats.adresregel2
+    En bevat het antwoord geen veld verblijfplaats.land
+    """
```

Comment:
Is het nog nodig om dit (de json) extra toe te voegen? In de `Dan ...` is al aangegeven dat verblijfplaats.vertrokkenOnbekendWaarheen de waarde true heeft

Reply:
verwijderd